docs/src/pages/guides/server-rendering/server-rendering.md

```diff
@@ -197,7 +197,7 @@ import getCache from './getCache';
 
 function Main() {
   return (
-    <CacheProvider value={getCache}>
+    <CacheProvider value={getCache()}>
```

Comment:
Is this really just a getter or does it also create the cache i.e. are we sure this is the same cache on every render?

Reply:
It's a factory, I have pushed a new commit to be consistent with the other examples, great point.